tests/phpunit/Logger/JsonLoggerTest.php

```diff
@@ -102,16 +104,17 @@ public function metricsProvider(): iterable
             $this->createCompleteResultsCollector(),
             [
                 'stats' => [
-                    'totalMutantsCount' => 12,
+                    'totalMutantsCount' => 14,
                     'killedCount' => 2,
                     'notCoveredCount' => 2,
                     'escapedCount' => 2,
                     'errorCount' => 2,
+                    'syntaxErrorCount' => 2,
                     'skippedCount' => 2,
                     'timeOutCount' => 2,
-                    'msi' => 60,
-                    'mutationCodeCoverage' => 80,
-                    'coveredCodeMsi' => 75,
+                    'msi' => 50,
+                    'mutationCodeCoverage' => 83.33,
+                    'coveredCodeMsi' => 60,
```

Comment:
Well, this is wrong. If Infection caused an arbitrary program to end with a syntax error, this was a killed mutant. Just like if mutation leads to a type error.